src/Driver.cpp

```diff
@@ -337,11 +337,10 @@ void DriverReportStartup(Driver* self, const char** targets, int target_count)
 
 bool DriverInitData(Driver* self)
 {
-  ProfilerScope prof_scope("Tundra InitData", 0);
-
   if (!DriverPrepareDag(self, s_DagFileName))
```

Comment:
do we not want to include this in any profile anymore?

Reply:
it moved down 5 lines.  tundra profiler does not support nested profiler scopes, so you have to be really careful to not nest them.  (and I added one in DriverPrepareDag)

Reply:
got it thanks